content/blog/Gone-phishing

```diff
@@ -0,0 +1,149 @@
+![ ](redirectlures.png)
+At the time of writing this the standard phishlet for office365 did not work, if you want to update the o365 phishlet here are the steps.
+
+Go to the directory where evilginx2 is located.
+```~/evilginx2 ls```
+``` cd phishlets```
+``` vim o365.yaml ```
+now you should be in the yaml file that holds the phishlet data.
+To delete the content easly hold ```shift + :``` to be able to type then type ```%d``` and press enter. normally the file should be empty now.
+Now go to this [link](https://gist.github.com/hazcod/887dc2bbc3eb90ac9fd7788867b23416) and copy the code.
+ Go back to your terminal and paste the code in the o365.yaml file and then again press ```shift + :``` and type ```wq``` followed by ```enter```.
+Start your evilginx2 up again and lets test our our phishing link.
+
+once in evilginx2 type
+```lures get-url 0```
+Copy the link and paste it in your browser.
+![ ](linktophish.png)
+To make sure that evilginx2 stays up and running after disconnecting you SSH connection we can put the evilginx2 session in [tmux](https://www.hamvocke.com/blog/a-quick-and-easy-guide-to-tmux/).
+To start up a new tmux session just type
+```tmux``` in your terminal and the session wil open.
+![ ](tmux1.png)
+Its best to split the tmux session in two so that you can multitask in your session without shutting down evilginx2.
+To do this press ```ctrl``` + ```B``` and then press ```%```
+you need to type % so if on your keyboard it is e.g shift + 5 then hold shift and % after the ctrl + B combination.
+![ ](tmux2.png)
+To switch between sides press ```ctrl``` + ```B``` and then press ```o```.
+now start evilginx2 in one side.
+
+To detach your current session use ```ctrl``` + ```B``` and then press ```d```.
+
+Now that your session is detached you can pick it up from where you left it at any later point in time. To re-attach to a session you need to figure out which session you want to attach to first. Figure out which sessions are running by using
+
+```tmux ls```
+This will give you a list of all running sessions, which in our example should be something like
+
+0: 1 windows (created Wed Oct  5 13:50:39 2022) (detached)
+To connect to that session you start tmux again but this time tell it which session to attach to:
+
+```tmux attach -t 0```
+Note that the -t 0 is the parameter that tells tmux which session to attach to. “0” is the first part of your tmux ls output.
+
+### Gophish setup
+To install gophish we can use the same way as with evilginx.
+Lets go fetch the [link](https://github.com/gophish/gophish/releases) that fits our OS architecture and download the zip file with wget on our VPS.
+![ ](copylink.png)
+In your VPS terminal
+```wget <link.to.zip>```
+``` unzip gophish-v0.12.1-linux-64bit.zip```
+You can run the gophish app in the same tmux version as your evilginx2. earlier we made a second screen inside the tmux version so lets go there (```tmux attach 0 -t```) go to the directory of gophish.
```

Comment:
`version`?
You actually don't in the same session, you best use two tmux sessions.
Also capital letters, use named tmux sessions.